jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735SupplementalVehicleExtensions.java

```diff
@@ -30,7 +30,7 @@
 	private J2735DisabledVehicle status;
 	private J2735SpeedProfile speedProfile;
 	private J2735RTCMPackage theRTCM;
-	private List<J2735RegionalContent> regional = new ArrayList<>();
+	// private List<J2735RegionalContent> regional = new ArrayList<>();
```

Comment:
Are we taking these out permanently? If so, let's remove instead of commenting.

Reply:
Okay, they are not really supported in J2735 so I don't know why they are used

Reply:
I will remove them

Reply:
Addressed